app/views/layouts/application.html.slim

```diff
@@ -32,11 +32,14 @@ html
         .row
           .span8
             p
-              | © 2012 OpenMensa
+              | © 2012-2015 OpenMensa
               | &nbsp;&middot;&nbsp;
               = link_to 'Impressum', imprint_path
               | &nbsp;&middot;&nbsp;
               = link_to 'Blog', 'http://blog.altimos.de/category/openmensa'
               | &nbsp;&middot;&nbsp;
               = link_to 'Developer', 'http://doc.openmensa.org/'
-
+              | &nbsp;&middot;&nbsp;
+              = link_to 'Über OpenMensa', about_path
+              | &nbsp;&middot;&nbsp;
+              = link_to 'Unterstüze uns', support_path
```

Comment:
My fear with "mach mit" it can be confused with "arbeite mit": I create one page for general support independent from any (programming skills) and one separate page for (programming) support.